src/RestControllers/AuthorizationController.php

```diff
@@ -694,14 +739,18 @@ private function verifyLogin($username, $password, $email = '', $type = 'api'):
         $auth = new AuthUtils($type);
         $is_true = $auth->confirmPassword($username, $password, $email);
         if (!$is_true) {
+            $this->logger->debug("AuthorizationController->verifyLogin() login attempt failed", ['username' => $username]);
             return false;
         }
         if ($this->userId = $auth->getUserId()) {
             $_SESSION['user_id'] = $this->getUserUuid($this->userId, 'users');
+            $this->logger->debug("AuthorizationController->verifyLogin() user login", ['pid' => $_SESSION['pid']]);
             return true;
         }
         if ($id = $auth->getPatientId()) {
             $_SESSION['user_id'] = $this->getUserUuid($id, 'patient');
+            $this->logger->debug("AuthorizationController->verifyLogin() patient login", ['pid' => $_SESSION['user_id']]);
+            $_SESSION['pid'] = $_SESSION['user_id'];
```

Comment:
was above line needed? seems odd since the oauth2 login stuff uses user_id session (worry here is setting this could open door to access in patient portal).

Reply:
@bradymiller see src/Common/Auth/OpenIDConnect/IdTokenSMARTResponse->getExtraParams() line 47-54.  We have to have the patient id for a standalone launch.  Since the AuthorizationController at this section of code is handling a user login we know the current user is a patient.  Is there a better way we can accomplish this functionality?

Essentially if a patient logs in and its a standalone app launch we need to be able to provide the patient id.

Reply:
Looked at code in more detail. Should be ok since it both requires patient login for this to be set and will be using the oauth cookie.